src/main/scala/dpla/ingestion3/mappers/providers/Extractor.scala

```diff
@@ -9,4 +10,14 @@ import dpla.ingestion3.model.{DplaMapData, EdmAgent}
 trait Extractor {
   def build(rawData: String): DplaMapData
   def agent: EdmAgent
-}
+  /**
+    * Generates an MD5 checksum from the given providerId
+    * Implemented per provider
+    *
+    * DigestUtils.md5Hex(id)
+    *
+    * @return MD5 hash value
+    * @throws NotImplemented
+    */
+  def mintItemId(rawData: Any): String
```

Comment:
Is there a better way to generically type the parameter? We want to send a NodeSeq or a JValue (possibly something else?). I've tried using `type T` and `Either[JValue, NodeSeq]` w/o success. 

The original issue I ran into was re-parsing the rawData twice when it was being passed in as a String (like it is done in `build()`) in each of the implementations and that looks pretty ugly. I also didn't want to repeat how the base ID was being selected. That should only exist in the provider implementation once.